build/build-darwin.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
While I appreciate the effort here, I was considering something more node-esque (since we will need to handle Windows). Here's a bunch. But before moving to any one of them, let's perform research and talk about it in #2:

- https://www.npmjs.com/package/gulp-atom-electron/
- https://www.npmjs.com/package/electron-packager (yours)
- https://www.npmjs.com/package/grunt-electron
- https://www.npmjs.com/package/grunt-electron-builder
- https://www.npmjs.com/package/grunt-electron-debian-installer